config/routes.rb

```diff
@@ -0,0 +1,58 @@
+Rails.application.routes.draw do
+  get 'main/index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.